duckduckgo/macos-browser
# Pass site parent entity to Privacy Dashboard

Task/Issue URL: https://app.asana.com/0/0/1201282361231436/f
Tech Design URL:
CC: @tomasstrba @mallexxx @samsymons  

**Description**:
Check tracker radar data for parent entity of the current domain. Pass this through to the Privacy Dashboard if found. This let's us flag up Tracker Network sites even if not tracking is occurring on that particular page.

Corresponding Privacy Dashboard PR: https://github.com/more-duckduckgo-org/duckduckgo-privacy-dashboard/pull/16
~⚠️ __Note:__ The Privacy Dashboard PR above must land first and this branch will need to be updated to point to latest `main` in `Submodules/duckduckgo-privacy-dashboard`~ -- Done

**Steps to test this PR**:
1. Build browser from this PR
1. Go to https://google.com
2. Confirm this site is flagged as Tracker Network, even though tracker count is 0

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
